test/falcor/error/index.js

```diff
@@ -18,19 +18,16 @@ describe("Error", function() {
         toObservable(model.
             get(["test", {to: 5}, "summary"])).
             doAction(noOp, function(err) {
-                expect(err.length).to.equal(6);
                 // not in boxValue mode
                 var expected = {
-                    path: [],
+                    $type: "error",
```

Comment:
These tests using ErrorDataSource are a little confusing, but looks like the `{$type: "error"...}` boxing is coming from the data source not the Model here.